### PR TITLE
Unit 6 - Dynamic content - Eddie Tong

### PR DESCRIPTION
**Pull request by Edison-Tong**
Screenshot 2024-08-03 at 6 50 02 PM

### CODE REVIEW

File: Unit 6/dynamic-content/quiz.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Dynamic Content Quiz
+
+1. How would yoyu change the text of all elements with the `class` "example" to "Example Text"?
+
+```javascript
+$(".example").attr("class", "Exmple-text");
```
Comment (AcquirEdge) [inline Unit 6/dynamic-content/quiz.md:6]: Not quite... We're changing text, not attributes here.

**Review by AcquirEdge — CHANGES_REQUESTED** [review]
just one quiz question needs a review. Otherwise looks good.

**Comment by Edison-Tong** [comment]
whoops sorry I misread that I think.

**Review by AcquirEdge — APPROVED** [review]
Looks good!

**Pull request merged**